includes/admin/settings/class.llms.settings.checkout.php

```diff
@@ -221,7 +217,7 @@ private function get_settings_default() {
 			array(
 				'default' => 'yes',
 				'desc'    => __( 'Enable automatic retry of failed recurring payments.', 'lifterlms' ) .
-							  '<p class="description">' . sprintf( __( 'Recover lost revenue from temporarily declined payment methods. %1$sLearn More%2$s', 'lifterlms' ), '<a href="https://lifterlms.com/docs/automatic-retry-failed-payments/" target="_blank">', '</a>' ) . '</p>',
+								'<p class="description">' . sprintf( __( 'Recover lost revenue from temporarily declined payment methods. %1$sLearn More%2$s', 'lifterlms' ), '<a href="https://lifterlms.com/docs/automatic-retry-failed-payments/" target="_blank">', '</a>' ) . '</p>',
 				'id'      => 'lifterlms_recurring_payment_retry',
 				'title'   => __( 'Retry Failed Payments', 'lifterlms' ),
```

Comment:
```suggestion
				'title'   => esc_html__( 'Retry Failed Payments', 'lifterlms' ),
```

Reply:
Not sure if the plugin checker is finding these here or elsewhere where a var is set to a translated string, but we basically can use esc_html__ to sanitize the value coming from the translators.